docs/tutorials/profile-example/ProfilePlugin.md

```diff
@@ -1,4 +1,4 @@
-# Implementing profiles in a Zowe CLI plug-in
+# Implementing profiles in a Zowe CLI Plug-in
```

Comment:
should plug-in be lowercase?

Reply:
I think profiles should be capitalized actually